u8timeseries/timeseries.py

```diff
@@ -804,10 +815,41 @@ def _combine_from_pd_ops(self, other: 'TimeSeries',
         conf_hi = self._combine_or_none(self._confidence_hi, other.conf_hi_pd_series(), combine_fn)
         return TimeSeries(series, conf_lo, conf_hi)
 
+    @staticmethod
+    def _fill_missing_dates(series: pd.Series) -> pd.Series:
+        """
+        Tries to fill missing dates in series with NaN.
+        Method is successful only when explicit frequency can be determined from all consecutive triple timestamps.
+
+        Parameters
+        ----------
+        series
+            The actual time series, as a pandas Series with a proper time index.
+
+        Returns
+        -------
+        pandas.Series
+            A new Pandas Series without missing dates.
+        """
+        date_axis = series.index
+        samples_size = 3
```

Comment:
do we have some sort of global variable for that? Would be easier to use it from one place until we start working on https://unit8.atlassian.net/browse/U8TS-66

Reply:
@TheMP Actually I'm not sure whether solving the issue you mentioned (which I did as part of the residuals pull request) is going to change much in that regard. As far as I can tell 3 is just the minimum series length to infer the frequency. Since Series instances shorter than 3 will be allowed, I will be sure to throw an exception if the frequency is not given explicitly. 
Sorry if this sounds confusing, once I'm done with my PR it should become clear when looking at the TimeSeries constructor.